html_app/build.py

```diff
@@ -0,0 +1,257 @@
+#!/usr/bin/env python
+
+"""The static asset pipeline for StarCellBio.
```

Comment:
Hadn't seen docops before, not sure it saves you much typing time over argparse, but it is a pretty neat concept. How do you specify integer/boolean/file args, or do nargs, or choices? It also is maybe a little too magical for me, but is fine to use here.